src/components/Card/Card.spec.js

```diff
@@ -9,5 +9,20 @@ describe('Card', () => {
     render(<Card name="Minikleid" image={imageFile} />)
     expect(screen.getByText('Minikleid')).toBeInTheDocument()
     expect(screen.getByRole('img')).toBeInTheDocument()
+    expect(screen.getByRole('button')).toBeInTheDocument()
+  })
+
+  it('Button calls Callback onClick when clicked', () => {
+    const callback = jest.fn()
+    render(
+      <Card
+        name="Minikleid"
+        image={imageFile}
+        onClick={callback}
```

Comment:
Please have a look at Victorias Pull request:
https://github.com/VictoriaWika/capstone-project/pull/18#discussion_r598459394
I explained why you don't need the `onClick` here.